### PR TITLE
add simple google streetview

### PR DESCRIPTION
**Pull request by mrtmrt233**
addresses #281 

simple hack: link in sidebar, open up streetview at tree location in a new window/tab

alternatively, we can do a overlay displaying streetview image on top of the current map (as proposed in the issue), but this would involve :
- an API key
- potential terms of service violation
- a lot more code

so maybe slate this for if and when necessary?

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -193,6 +198,10 @@ <h5>Location</h5>
           <h4>NEAREST ADDRESS:</h4>
           <span id="sidebar-address"></span>
           <button id="sidebar-details-button" class = "btn btn--mobile">View Details</button>
+          <div id="google-streetview"> 
+            <h4>
+            <a href="#" id = "gsv-link" target="_blank" rel="noopener noreferrer">Google Street View</a> </h4> 
+          </div>
```
Comment (emillipede) [inline public/index.html:204]: @mrtmrt233 I think I've somehow messed up this function. Now when I test it, the link doesn't open street view in a new tab. I'm sorry, I don't know what I did :/

File: public/js/Sidebar.js [hunk 2]
```diff
@@ -59,10 +59,12 @@ var initialY = null;
     this.vacantReplacementSpecies      = document.getElementById('sidebar-vacant-replacement-species');
     this.vacantStreetSegment           = document.getElementById('sidebar-vacant-street-segment');
     this.vacantCloseButton             = document.getElementById('sidebar-vacant-close-button');
-    this.vacantDetailsButton           = document.getElementById('sidebar-vacant-details-button')
+    this.vacantDetailsButton           = document.getElementById('sidebar-vacant-details-button');
```
Comment (emillipede) [inline public/js/Sidebar.js:63]: (I'm trying to fix the problem I caused)

**Comment by mrtmrt233** [comment]
should be working now

**Comment by emillipede** [comment]
thank you @mrtmrt233, I'm sorry I messed with the PR.

**Pull request merged**